pdns/signingpipe.cc

```diff
@@ -1,3 +1,4 @@
+#include "threadname.hh"
```

Comment:
its weird to do this before config.h, but i guess its like that in the original pr

Reply:
ah yes. Guess my editor (with help from clangd) did that. No ifdefs in the `.hh` so we get away with it.